docs/labs.md

```diff
@@ -49,3 +49,9 @@ That's it. Now should see your new counter under the header.
 ## Multiple integration managers (`feature_many_integration_managers`)
 
 Exposes a way to access all the integration managers known to Riot. This is an implementation of [MSC1957](https://github.com/matrix-org/matrix-doc/pull/1957).
+
+## New ways to ignore people (`feature_mjolnir`)
+
+When enabled, a new settings tab appears for users to be able to manage their ban lists. 
```

Comment:
Could you add some link to mjolnir documentation or explain the idea of ban lists here?